src/core/optimization_container.jl

```diff
@@ -640,35 +646,51 @@ function get_problem_size(optimization_container::OptimizationContainer)
     return "The current total number of variables is $(vars) and total number of constraints is $(cons)"
 end
 
-function build_impl(
+function build_impl!(
```

Comment:
Is `build_impl` short hand for build implementation? What do you think about using `build_problem!` here instead as the name of the function?

Reply:
I wanted to avoid confusion when reading the code between our internal method and the ones user's would need to overload.